client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -16,10 +16,10 @@ import {
   archiveItem,
   clearVote,
   cloneItem,
-  cloneItemWS,
+  itemBusy,
```

Comment:
itemBusy doesn't sound like an action/method - maybe setItemBusy, the same for itemFree